.github/workflows/nightly-rockset-uploads.yml

```diff
@@ -33,4 +33,28 @@ jobs:
           retry_wait_seconds: 90
           command: |
             echo "Uploading external contribution stats for" "$(date -d yesterday '+%Y-%m-%d')"
-            python3 -m tools.stats.upload_external_contrib_stats --startDate "$(date -d yesterday '+%Y-%m-%d')"
\ No newline at end of file
+            python3 -m tools.stats.upload_external_contrib_stats --startDate "$(date -d yesterday '+%Y-%m-%d')"
+
+  upload-testing-stats-aggregates:
```

Comment:
Is there any particular reason why `upload-contribution-stats` and `upload-testing-stats-aggregates` need to be run on separate runners?  It looks like they could be combined.

Reply:
They don't at all